scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -663,7 +663,7 @@ class FormatWriter(formatOps: FormatOps) {
     val units = Vector.newBuilder[AlignmentUnit]
     while (i < block.length) {
       val line = block(i)
-      if (line.lengthCompare(column) == 1) {
+      if (column < line.length) {
```

Comment:
also, if you like `lengthCompare`, `line.lengthCompare(column) > 0`, although not necessarily more readable :)

Reply:
When I originally wrote `lengthCompare` I overlooked that `line` is `Array`. So `lengthCompare` is redundant optimization here :)